### PR TITLE
feat: add empty room and corridor models

### PR DESCRIPTION
**Pull request by SSelinn**
All models of corridor and empty room are in the prefab folder. The teleport room is scrapped. We can use a DeadEnd room and have a teleport item that we can later spawn where ever we want to have better control over where it is located.

### CODE REVIEW

**Review by JoenvandeVorle — CHANGES_REQUESTED** [review]
Models look nice, only thing is you should add the `Ground` layer to them so the player can actually walk on it

File: aplib.net-demo/Assets/Prefabs/Empty Room.prefab
Comment (QuakeEye) [inline aplib.net-demo/Assets/Prefabs/Empty Room.prefab:1]: Please rename this to `EmptyRoom`, spaces in names can break stuff later on.

File: aplib.net-demo/Assets/Prefabs/Empty Room.prefab.meta
Comment (QuakeEye) [inline aplib.net-demo/Assets/Prefabs/Empty Room.prefab.meta:1]: Please rename this to `EmptyRoom`, spaces in names can break stuff later on.

File: aplib.net-demo/Assets/Prefabs/Corner.prefab
Comment (QuakeEye) [inline aplib.net-demo/Assets/Prefabs/Corner.prefab:1]: As an overal comment for all prefabs (so handle this for all of them), please add the proper layer so the player can interact with them (in this case, at least for now, layer `Ground`)

Comment (QuakeEye) [inline aplib.net-demo/Assets/Prefabs/Corner.prefab:1]: As an overal comment for all prefabs (so handle this for all of them), please do not put them in the root of the prefab folder, make some path such as `Assets/Prefabs/Environment/ProBuilderPieces` or something like that

**Review by QuakeEye — CHANGES_REQUESTED** [review]

**Review by QuakeEye — APPROVED** [review]

**Review by QuakeEye — APPROVED** [review]

**Review by JoenvandeVorle — APPROVED** [review]
spime (seems perfect in my eyes)

**Pull request merged**